Pycloud Deployer/config.py

```diff
@@ -0,0 +1,30 @@
+import os
+import logging
+from dotenv import load_dotenv
+
+logger =  logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
You've set the logging level to DEBUG for the entire logger, which is fine for development but might be too verbose for a production environment. Consider making the logging level configurable through an environment variable or a configuration file.